app/assets/stylesheets/application.scss

```diff
@@ -3,22 +3,23 @@
 // govuk_frontend_toolkit imports
 @import 'colours';
 @import 'measurements';
-@import 'typography';
 @import 'shims';
+@import 'typography';
 
 // helpers for common page elements
-@import "helpers/responsive-bottom-margin";
```

Comment:
Hi,

I think we're getting away with it because notice (and description should suffer too) are `@mixin`s and aren't getting evaluated until they're used. The first time it's used is the first view import `@import "views/case-studies";`.

The tests pass and it works in the flesh but I think it would read better as it was ordered originally. I'll add a PR to reorder them again.

Thanks